tests/Yaapii.Atoms.Tests/Map/GroupedTest.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Xunit;
+using Yaapii.Atoms.Enumerable;
+using Yaapii.Atoms.Func;
+using Yaapii.Atoms.IO;
+using Yaapii.Atoms.List;
+using Yaapii.Atoms.Map;
+using Yaapii.Atoms.Number;
+
+namespace Yaapii.Atoms.Map.Tests
+{
+    public sealed class GroupedTest
+    {
+        [Fact]
+        public void GroupsListToDictionaryOfKeysAndLists()
+        {
+            var srcList = new ListOf<string>("ABC", "ABCD", "ABCDE");
+            var keyFunc =
```

Comment:
its not a keyFunc - its the key (in the shape of a func). func is encoding in names. If you want to make it clear that its a procedure, name it getsKey for example (its "someone" who gets a key)